keras/ops/core.py

```diff
@@ -661,10 +662,19 @@ def custom_gradient(f):
     def log1pexp(x):
         e = ops.exp(x)
 
-        def grad(upstream):
+        def grad(*args, upstream = None):
+            if upstream == None:
+                upstream, = args
+                
             return ops.multiply(upstream, 1.0 - 1.0 / ops.add(1, e))
 
         return ops.log(1 + e), grad
     ```
+
+    Note that the grad function that returns gradient computations 
```

Comment:
To make this clearer, please provide two separate code examples, one for JAX/TF and one for torch.

Reply:
> To make this clearer, please provide two separate code examples, one for JAX/TF and one for torch.

Added one JAX/TF-specific backend example, one PyTorch and one for all three together (backend-invariant).
Can you check if my edited PRs are visible to you or not?